modules/core/shared/src/main/scala/lucuma/core/math/Constants.scala

```diff
@@ -4,27 +4,49 @@
 package lucuma.core.math
 
 import java.time.Duration
+import coulomb._
+import coulomb.physicalconstants._
+import lucuma.core.math.units.MetersPerSecond
+import spire.math.Rational
+import coulomb.define.DerivedUnit
+import coulomb.si.Meter
+import coulomb.physicalconstants.infra.PhysicalConstantQuantity
 
 object Constants {
+  object infra {
+    trait AstronomicalUnit
```

Comment:
Can we make this a `type` alias so that it doesn't generate a `.class` ?

Otherwise, can it be `sealed` ?

Reply:
Maybe I'll make object `infra` private, these definitions shouldn't really be available outside, you want to only see the `def`s.

Reply:
I can replace it with `type` and it compiles fine, `coulomb` uses a `trait` in all these cases so I'm a bit reluctant to change it unless there is a clear benefit